From f31131c81bf75d70a6a8d93e33331d3a66eb5b63 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 26 Sep 2018 21:50:02 +0100
Subject: [PATCH 5/5] core_base: Remove libnfc_ndef from PRODUCT_PACKAGES

 * This was a dependency of legacy NXP NFC stack, deprecated long ago.
   The devices that are still using this stack are explicitly building
   libnfc, which in turn has libnfc_ndef as shared lib.

Change-Id: Id3b859cdea00452f7d1ee21455e08fe342649a1e
---
 target/product/core_base.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/core_base.mk b/target/product/core_base.mk
index 8a0515a95..5b37b8de7 100644
--- a/target/product/core_base.mk
+++ b/target/product/core_base.mk
@@ -29,7 +29,6 @@ PRODUCT_PACKAGES += \
     libfilterpack_imageproc \
     libgabi++ \
     libmdnssd \
-    libnfc_ndef \
     libpowermanager \
     libspeexresampler \
     libstagefright_soft_aacdec \
-- 
2.17.1

